flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is still disappointing.

Reply:
I don't know how to do it

Reply:
how do you build it?

FFMPEG is build on so many project on flathub that there is absolutely no secrets.

Reply:
I don't want to build it. I just need to download a different build

Reply:
you should be building it.

Reply:
Does flathub buildbot have docker?

Reply:
docker is not a build system.

Reply:
The custom ffmpeg that I am using requires docker to build

Reply:
I think I finally figured it out. I am feeling stupid that I didn't think of that before. All I had to do was download both ffmpeg binaries, then check the processor arch and use the required binaries